site/docs/hosting/deno-deploy.md

```diff
@@ -59,42 +59,44 @@ Here, we are using the bot token (`/<bot token>`).
 
-3. Visit your [Deno Deploy dashboard](https://dash.deno.com/projects).
-4. Click on "New Project", and go to the "Deploy from GitHub repository" section.
+3. Visit your [Deno Deploy dashboard](https://dash.deno.com/account/overview).
+4. Click on "New Project".
 5. Install the GitHub app on your account or organization, and choose your repository.
-6. Select the branch you want to deploy, and then choose your `mod.ts` file to be deployed.
+6. Select the branch you want to deploy.
+7. Select the entrypoint file `main.ts`, and click "Deploy Project" to deploy.
 
 ### Method 2: With `deployctl`
 
-> This is a method for more advanced users. It allows you to deploy the project via the command line or Github Actions.
+> This is a method for more advanced users or if you don't want to upload your code to GitHub.
+> It allows you to deploy the project via the command line or GitHub Actions.
 
-1. Visit your [Deno Deploy dashboard](https://dash.deno.com/projects).
-2. Click "New Project", and then "Empty Project".
-3. Install [`deployctl`](https://github.com/denoland/deployctl).
-4. [Create an access token](https://dash.deno.com/account#access-tokens).
-5. Run the following command:
-
-   ```sh
-   deployctl deploy --project <project> ./mod.ts --prod --token <token>
+1. Install [`deployctl`](https://github.com/denoland/deployctl).
+2. Create an access token from the "Access Tokens" section in [account settings](https://dash.deno.com/account).
+3. Go to your project directory and run the following command:
+   ```sh:no-line-numbers
+   deployctl deploy --project=<project> --entrypoint=./main.ts --prod --token=<token>
    ```
 
-6. To set up Github Actions, refer to [this](https://github.com/denoland/deployctl/blob/main/action/README.md).
+   ::: tip Setting environment variables
+   Environment variables can be set by heading to your project's settings after deploying.
+
+   But this is possible from the command line, as well:
 
-### Method 3: With URL
+   1. You can assign environment variables from a dotenv file by adding the `--env-file=<file>` argument.
+   2. You can also specify them individually by using the `--env=<key=value>` argument.
 
-> All you need for following this method to deploy your grammY bot, is a public URL to your `mod.ts` file.
+   :::
+4. To set up GitHub Actions, refer to [this](https://github.com/denoland/deployctl/blob/main/action/README.md).
 
-1. Create a new project on Deno Deploy.
-2. Click "Deploy URL".
-3. Input the public URL to your `mod.ts` file, and click "Deploy".
+Check out the [deployctl documentation](https://docs.deno.com/deploy/manual/deployctl) for more information.
 
 ### Note
 
 After getting your app running, you should configure your bot's webhook settings to point to your app.
 To do that, send a request to
 
-```text
-https://api.telegram.org/bot<token>/setWebhook?url=<url>
+```http:no-line-numbers
+GET https://api.telegram.org/bot<token>/setWebhook?url=<url>
```

Comment:
What do you think of making this a curl command instead?

Reply:
Hmm, in a way it makes sense, and probably handy for beginners if they are wondering how to make this request.

But thinking the opposite, wouldn't it sound like we are suggesting that they _should_ run the command? I thought this part implied that they should just make this request, by using any method. I usually do this right in the browser.

I'm up for both, and probably overthinking. What do you think, should we change it?

Reply:
No, let's leave it. Then we can discuss in https://t.me/grammyjs if we should use HTTP notation or a curl command, and follow up with a separate PQ that perform this change consistently across all hosting guides.

Reply:
https://t.me/grammyjs/248240

Reply:
Should I change to curl then? According to the votes?

Reply:
Yep sounds good

Reply:
Done; sorry it took me a while to get back to this.